.github/CODEOWNERS

```diff
@@ -0,0 +1,50 @@
+###################################################################################################
+# Overview
+#
+# Pattern used to match files follows most of the same rules as used in gitignore files. Order is
+# important; the last matching pattern takes precendence.
+#
+# For more info see:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+###################################################################################################
+
+
+###################################################################################################
+# Global (@keycloak/maintainers)
+###################################################################################################
+
+/**                                                         @keycloak/maintainers
+
+###################################################################################################
+# Testing (@keycloak/testing-maintainers)
+###################################################################################################
+
+/testsuite/**                                               @keycloak/testing-maintainers
+
+###################################################################################################
+# Core (@keycloak/core-maintainers)
+###################################################################################################
+
```

Comment:
Nothing for core? I guess this is hard to define in the current directory structure?

Reply:
TBD ;)